tests/unit/lib/foxbox/services_test.js

```diff
@@ -0,0 +1,195 @@
+import { waitFor } from '../../test-utils';
```

Comment:
Here not all cases covered, just most important and obvious. Should be good enough to start with.